mem-store.ts

```diff
@@ -1,8 +1,7 @@
 /* Copyright (c) 2010-2020 Richard Rodger and other contributors, MIT License */
 'use strict'
 
-import Assert from 'assert'
-import * as Common from './lib/common'
+import * as Intern from './lib/intern'
```

Comment:
let's use the form:
`import { intern } from './lib/intern'`

and also establish the convention to use `intern` not `Intern`

this is because some repos have a small intern set and do not use a separate file
it also indicates the intern is special - not really a library

Reply:
@rjrodger 
Woah, that actually circumvents the issue with the misreported coverage?
```
Coverage: 91.73% (31/375)
dist/mem-store.js missing coverage from file(s):
        ../mem-store.ts on line(s): 127, 132, 146, 147, 151, 176, 193, 193, 208, 221, 228, 245, 246, 249, 260, 26
1, 266, 270, 277, 278, 298, 303, 330, 331, 363
dist/lib/intern.js missing coverage from file(s):
        ../../lib/intern.ts on line(s): 114, 123, 128, 141, 152, 175
```